src/django_components/compat/django.py

```diff
@@ -114,3 +121,44 @@ def handle(self, *args: Any, **options: Any) -> None:
     Command.__doc__ = command.__doc__
 
     return Command
+
+
+################################################
+# ROUTING
+################################################
+
+
+def routes_to_django(routes: List[URLRoute]) -> List[URLPattern]:
```

Comment:
Same way as there is a function that converts our own CLI command format into Django's, here is a fuction that converts our own URL route format into Django's.